src/main/resources/static/css/styles.css

```diff
@@ -1,6 +1,13 @@
 body {
-  font-family: Arial, Helvetica, sans-serif;
+  font-family: 'Source Sans Pro', Arial, Helvetica, sans-serif;
   font-size: 16px;
+  background: rgb(250, 250, 250);
+}
+
+h1 {
```

Comment:
please don't style html tags. Name it, e.g. `class="products-header"` and style the class.